### DISCUSSION TITLE
Tailwind for media queries alone.

### DISCUSSION THREAD

**Discussion by chinedu360** [tailwindlabs/tailwindcss#5049] [Help] [answered]
Can tailwind be used for media queries on a next.js react app built with styled component?

**Comment by dance2die** [comment, marked as answer]
TW (Tailwind CSS) provides breakpoints (defaults found [here](https://tailwindcss.com/docs/breakpoints)), which you'd use with breakpoint variants (e.g. `sm:`, `md:`, `lg:`).

So using mobile first approach, `<div className="text-sm lg:text-2xl>...</div>` will have small text sizes by default and will have 2xl font size when you hit the `lg` (large) breakpoint.

OFC, you can customize media query breakpoints. https://tailwindcss.com/docs/breakpoints#custom-media-queries.

Lastly, as you use Styled Components, you might want to check out [twin.macro](https://github.com/ben-rogerson/twin.macro#how-it-works).

**Reply by chinedu360** [reply]
ok thank @dance2die
